libraries/AP_EFI/AP_EFI.h

```diff
@@ -92,6 +93,11 @@ class AP_EFI {
     // send EFI_STATUS
     void send_mavlink_status(mavlink_channel_t chan);
 
+#if AP_SCRIPTING_ENABLED
```

Comment:
If EFI_SCRIPTING is the only user of this method then perhaps use AP_EF_SCRIPTING_ENABLED here instead to save bytes where scripting isn't enabled.

Reply:
method removed